mozmeao/MIECO
# Convert landing page to use jinja templates

## What this does
- Convert landing page to use jinja templates
- Create a jinja partial for the teams section due to how long it is
- update build.py to ignore files in the `templates/` and `partials/` directories
- Added scripts to run `build.py` on file change and on `npm run build`

The python scripts will output needed html to `html_output/` and the parcel script will read from that directory

## Testing
-install python deps with `pip install -r requirements.txt`
- install node with `npm install`
- run `npm run start` and localhost:8000 should match what is on https://future.mozilla.org/mieco
